src/classes/OPP_AutomatedSoftCreditsService.cls

```diff
@@ -38,6 +38,10 @@
 public class OPP_AutomatedSoftCreditsService {
 
     private static final String RELATED_OPPORTUNITY_CONTACT_ROLE_NONE = '--None--';
+    private static final String OBJECT_TYPE_RELATIONSHIP = 'npe4__Relationship__c';
```

Comment:
I'm fine with using the strings, but another approach (and one that we used with Customizable Rollups) is to work entirely with SObjectTypes & Field Tokens:
```
private static final SObjectType OBJECT_TYPE_RELATIONSHIP = npe4__Relationship__c.SObjectType;
private static final SObjectType OBJECT_TYPE_AFFILIATION = npe5__Affiliation__c.SObjectType;
private static final SObjectField FIELD_NAME_CONTACT = npe5__Affiliation__c.npe5__Contact__c;
private static final SObjectField FILED_NAME_RELATED_CONTACT = npe4__Relationship__c. npe4__RelatedContact__c;
```

Reply:
Cool, I'll look into that.